pyccel/ast/sympy_helper.py

```diff
@@ -128,6 +128,12 @@ def pyccel_to_sympy(expr, symbol_map, used_names):
     elif isinstance(expr, LiteralFloat):
         return sp.Float(expr)
 
+    if isinstance(expr, int):
+        return sp.Integer(expr)
+
+    elif isinstance(expr, float):
+        return sp.Float(expr)
```

Comment:
do you need this conversion I thought that all const values in the code are instances of Literal?

Reply:
some cases they are instances of python integer

Reply:
Do you remember which cases had integers? Probably it is there where there is a problem rather than here. We need to try and eliminate those bugs so we print with correct precision